docs/source/cugraph_intro.md

```diff
@@ -1,8 +1,72 @@
+relationship between data elements. In those cases, the data is best 
+represented as a graph. Graph analysis, also called network analysis, 
+is a collection of algorithms for answering questions posed against 
+graph data. Graph analysis is not new.
 
+The first graph problem was posed by Euler in 1736, the [Seven Bridges of 
+Konigsberg](https://en.wikipedia.org/wiki/Seven_Bridges_of_K%C3%B6nigsberg), 
+and laid the foundation for the mathematical field of graph theory. 
+The application of graph analysis covers a wide variety of fields, including 
+marketing, biology, physics, computer science, sociology, and cyber to name a few.
 
-## Terminology
+RAPIDS cuGraph is a library of graph algorithms that seamlessly integrates 
+into the RAPIDS data science ecosystem and allows the data scientist to easily 
+call graph algorithms using data stored in a GPU DataFrame, NetworkX Graphs, or even 
+CuPy or SciPy sparse Matrix.  
+
+
+# Vision
+The vision of RAPIDS cuGraph is to ___make graph analysis ubiquitous to the 
+point that users just think in terms of analysis and not technologies or 
+frameworks___. This is a goal that many of us on the cuGraph team have been 
+working on for almost twenty years. Many of the early attempts focused on 
+solving one problem or using one technique. Those early attempts worked for 
+the initial goal but tended to break as the scope changed (e.g., shifting 
+to solving a dynamic graph problem with a static graph solution). The limiting 
+factors usually came down to compute power, ease-of-use, or choosing a data 
+structure that was not suited for all problems. NVIDIA GPUs, CUDA, and RAPIDS 
+have totally changed the paradigm and the goal of an accelerated unified graph 
+analytic library is now possible.
+
+The compute power of the latest NVIDIA GPUs (RAPIDS supports Pascal and later 
+GPU architectures) make graph analytics 20x faster on average over NetworkX. 
+Moreover, the internal memory speed within a GPU allows cuGraph to rapidly 
+switch the data structure to best suit the needs of the analytic rather than 
+being restricted to a single data structure. cuGraph is working with several 
+frameworks for both static and dynamic graph data structures so that we always 
+have a solution to any graph problem. Since Python has emerged as the de facto 
+language for data science, allowing interactivity and the ability to run graph 
+analytics in Python makes cuGraph familiar and approachable. RAPIDS wraps all 
+the graph analytic goodness mentioned above with the ability to perform 
+high-speed ETL, statistics, and machine learning. To make things even better, 
+RAPIDS and DASK allows cuGraph to scale to multiple GPUs to support 
+multi-billion edge graphs.
+
+Every release of RAPIDS is accompanied with one or more wonderful BLOG(s) 
+about the features of that release (for example, see the release 0.8 blog [1] 
+or Life after Hadoop). This article is slightly different. While current 
```

Comment:
Sounds like this was taken from a blog

Reply:
it was :-)

Reply:
Removed that paragraph since it was unneeded